docs/verification/horei-verification.md

```
@@ -935,6 +935,36 @@ AIによる下書き照合。#194(法令上の制限2周目トラッキング)/
 
 ---
 
+## 追加照合(#197 分・2026-08-21)— q93 国土利用計画法(事後届出)2周目
+
+#194(法令上の制限2周目トラッキング)/ #197 の子イシューとして、q25 事後届出の2問目(別シナリオ)を追加した。利用者(hiiragi17)からe-Gov法令検索の逐語テキスト(14条1項・23条1項・23条2項1号柱書・イ・ロ・ハ)の提供を受け、下記のとおり原文で確認した。版はq25と同じ `349AC1000000092_20250601_504AC0000000068`(令和7年6月1日施行)で、利用者が2026-08-21時点のe-Gov上で確認したところ現在もこの版が現在施行(これより新しい版が無い)。
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)